notebooks/116-sparsity-optimization/116-sparsity-optimization.ipynb

```diff
@@ -8,7 +8,9 @@
    },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    !pip install optimum[openvino] openvino-dev datasets
</pre><p>openvino-dev installed as part regular dependencies of openvino_notebooks repo, please remove it from this line.</p><p>Also please use -q parameter for pip for avoid large installation log</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/903/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Thanks for the tip. Added in the new commit.